lib/measured/units/volume.rb

```diff
@@ -0,0 +1,15 @@
+Measured::Volume = Measured.build do
+  si_unit :l, aliases: [:liter, :litre, :liters, :litres]
+
+  unit :m3, value: "1000 l", aliases: [:cubic_meter, :cubic_metre, :cubic_meters, :cubic_metres]
+  unit :ft3, value: "0.0283 m3", aliases: [:cubic_foot, :cubic_feet]
+  unit :in3, value: "0.000578704 ft3", aliases: [:cubic_inch, :cubic_inches]
+  unit :gal, value: "0.00454609 m3", aliases: [:imp_gal, :imperial_gallon, :imp_gals,:imperial_gallons]
+  unit :us_gal, value: "0.00378541 m3", aliases: [:us_gallon, :us_gals, :us_gallons]
```

Comment:
I feel like we could find nicer representations of these. For example, when I googled `1 us gallon in gallons` I found this:

> The US gallon is used in the United States and is equal to exactly 231 cubic inches

I'd prefer doing that to allow the conversion table code to build optimal conversions between various units :)

Reply:
Also, sorry for the delay! Things are quite busy around here 🐌